content/en/tracing/trace_collection/admission_controller.md

```diff
@@ -89,6 +85,8 @@ To select your pods for library injection, annotate them with the following, cor
 
 The available library versions are listed in each container registry.
 
+**Note**: As mentioned above, library injection is applied on new pods only and does not have any impact on running pods. However, if you already have an application instrumented using version X of the library, and then use library injection to instrument using version Y of the same tracer library, the tracer will not break. Rather, the library version to be first loaded will be used.
```

Comment:
```suggestion
**Note**: If you already have an application instrumented using version X of the library, and then use library injection to instrument using version Y of the same tracer library, the tracer will not break. Rather, the library version to be first loaded will be used.
```
In your example with vX used in the application and lib injection on vY, which one would be used in the end? The current statement is not explicit

Reply:
It would most likely be Y. I would assume that the customer would use Lib Injection for their latest setup.